docs/AboutUs.md

```diff
@@ -1,59 +1,51 @@
----
-layout: page
-title: About Us
----
+# About Us
 
 We are a team based in the [School of Computing, National University of Singapore](http://www.comp.nus.edu.sg).
 
 You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ## Project team
 
-### John Doe
+### Faqih
 
-<img src="images/johndoe.png" width="200px">
+<img src="images/abdurrahmanfaqihiskandar.png" width="200px">
 
-[[homepage](http://www.comp.nus.edu.sg/~damithch)]
-[[github](https://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[github](https://github.com/abdurrahmanfaqihiskandar)][[portfolio](team/johndoe.md)]
```

Comment:
I think can remove all the portfolio section. Since it's all johndoe.md

Reply:
yup done